### ISSUE TITLE
Dominic Sicilian

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#34]

**Comment by aflies1** [comment]
Louis Technical Comments: "worked on 4 different projects in data science during his reasearch. MCMC (monte carlo markov chains) and nested fitting. He used multi layered neural networks as well. He got so so results because he was using x ray data that are notoriously low stat accuracy. They ran a model on the results  from the model to discard the poor results and got the accuracy up to 90%.

He has experince using very large datasets, and he thus is familiar with that type of huge dataset (several TB datasets). In his black matter project, he had to come up with a method to filter out the instrument background to be able to pick up a very sensitive feature that is the black mnatter impact, and he came up with a pipeline to filter the data  and remove the instrument background from it.

He only used python to handle the data, and not SQL. He only started to use SQL late, and his skills set is pretty nascent.

TL;DR: he has a very deep technical understanding of the models he has created, coming from a science heavy field. On the other hand, he hasn't heard of neither XGBoost nor Kaggle, which every data scientist would have. But I think that it is much easier to go from the technical details of some models and climbing on the superficial understanding required to create models as we do in our projects than the other way around. The other minus is the small SQL background, especially with USAA that is very SQL heavy, but again,  he seems very sharp and I think he'd be able to hop on it quickly. His python skills are very high which would have been a ton of work to get on top off so that's really good."

**Comment by aflies1** [comment]
Adam Comments:
Did his research on ProMazo and was very excited about the model that we created. Vert extensive background in Astrophysics, where he has collected worked with large data sets. A lot o fhis academic work has been team based, so he has either been working with professors, grad students, or undergraduates. 

He seems very interested in working with young talent and getting involved with a company that is doing something unique. He has never seen a company quite like ours and mentioned it was exactly what he was looking for in his next steps in his career.

I'd want to have another conversation with him to go deeper in a few areas but I think he is someone we should target as a PM if we think he can handle the technical work and client engagement.

**Comment by aflies1** [comment]
https://drive.google.com/file/d/1sGkT5Uvdj-J94Pt_Hkl0rXvo2r4JJkyr/view?usp=sharing
